plugins/woocommerce/client/blocks/tests/e2e/tests/checkout/checkout-block.merchant.block_theme.spec.ts

```diff
@@ -147,72 +147,72 @@ test.describe( 'Merchant → Checkout', () => {
 			).toBeVisible();
 		} );
 	} );
-	test( 'Merchant can see T&S and Privacy Policy links with checkbox', async ( {
-		frontendUtils,
-		checkoutPageObject,
-		admin,
-		editor,
-	} ) => {
-		await admin.visitSiteEditor( {
-			postId: 'woocommerce/woocommerce//page-checkout',
-			postType: 'wp_template',
-			canvas: 'edit',
-		} );
-		await editor.openDocumentSettingsSidebar();
-		await editor.selectBlocks(
-			blockSelectorInEditor +
-				'  [data-type="woocommerce/checkout-terms-block"]'
-		);
-		let requireTermsCheckbox = editor.page.getByRole( 'checkbox', {
-			name: 'Require checkbox',
-			exact: true,
-		} );
-		await requireTermsCheckbox.check();
-		await editor.saveSiteEditorEntities();
-		await frontendUtils.goToShop();
-		await frontendUtils.addToCart( REGULAR_PRICED_PRODUCT_NAME );
-		await frontendUtils.goToCheckout();
-		await checkoutPageObject.fillInCheckoutWithTestData();
-		await checkoutPageObject.placeOrder( false );
-
-		const checkboxWithError = frontendUtils.page.getByLabel(
-			'You must accept our Terms and Conditions and Privacy Policy to continue with your purchase.'
-		);
-		await expect( checkboxWithError ).toHaveAttribute(
-			'aria-invalid',
-			'true'
-		);
 
-		await frontendUtils.page
-			.getByLabel(
+	// This test is flaky. Skipping for now.
```

Comment:
Should we open an issue for this?

Reply:
Yeah, some of those should be addressed already by https://github.com/woocommerce/woocommerce/pull/55989. I will rebase and see if these skips are still needed.

Reply:
Follow up: p1742291538937259/1741293577.344589-slack-C03CPM3UXDJ